transport-native-io_uring/src/main/c/netty_io_uring.h

```diff
@@ -38,9 +38,6 @@ struct io_uring_sq {
     unsigned *array;
     struct io_uring_sqe *sqes;
 
-    unsigned sqe_head;
-    unsigned sqe_tail;
```

Comment:
why this change ?

Reply:
we don't use and we only need io_uring_sq struct(which is on the stack) for storing kernel pointers for the RingBuffer creation